network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/TerminalBusBreakerViewImpl.java

```diff
@@ -146,9 +146,13 @@ public void moveConnectable(String busId, boolean connected) {
             throw new PowsyblException("Trying to move connectable " + attributes.getResource().getId()
                     + " to bus " + busId + " of voltage level " + bus.getVoltageLevel().getId() + ", which is a node breaker voltage level");
         }
-        attributes.setConnectableBus(busId);
-        attributes.setBus(connected ? busId : null);
-        attributes.setVoltageLevelId(voltageLevel.getId());
+        getAbstractIdentifiable().updateResource(res -> {
+            InjectionAttributes attr = attributesGetter.apply(res);
+            attr.setConnectableBus(busId);
+            attr.setBus(connected ? busId : null);
+            attr.setNode(null);
```

Comment:
could you explain the reason to set null value for node in this case ?

Reply:
Because an equipment may be connected by Node (in node breaker view) of by bus (in bus breaker view). When you affect one of those values the other makes no sense anymore (and this may cause problems of you reaffect to the previous value, like in the unit tests).

However this brings a good question : may an equipment be connected to a bus and a node **in the same voltage level** ? In the unit test the bus breaker view and the node breaker view were on different voltage levels so the previous value had to be nulled.

I don't really know the answer, and I will ask to someone who knows about this subject.